tests/tracer/runtime/test_runtime_metrics.py

```diff
@@ -131,12 +131,12 @@ def test_only_root_span_runtime_internal_span_types(self):
                 assert root.get_tag("language") == "python"
                 assert child.get_tag("language") is None
 
-    def test_span_no_runtime_tags(self):
+    def test_span_always_runtime_tags(self):
```

Comment:
I am not sure we even need this test case anymore, since we always have the language tag on spans, we could remove this test and instead add one for the trace tags processor to ensure it always sets it on the chunk root span.

Reply:
Did this.